transcom/mymove
# Update email: ppm has been processed

## From Agility Ticket [B-21525](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21525)

## Testing Steps I took
1. Start the local server with the following command:
```
EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run
```
2. Start the local client.
3. As a Service Member, create a new PPM move and send to Service Counsellor. Make sure you try this at least once with one of each of the following categories:
- Marines, Navy, or Coast Guard
- Army
- Air Force or Space Force

_Note: Make sure you use an email address that you have access to and have a checkmark in the "Email" checkbox for preferred contact method._

4. As Service Counsellor, update the orders and approve the move and be sure to try at least one time with Actual Expense Reimbursement checked and at least one time with it unchecked.
5. As the Service Member, make sure to finish the process to that it shows up in the PPM Closeout Queue for the Service Counsellor role.
6. As a Closeout Counsellor, make sure you can see the move under the PPM Closeout tab and open it. You may have to click on the "Review Documents" button and approve any of the items that need approval.
7. Open your email and you should see several emails.. one of which will have the subject:
```
Your Personally Procured Move (PPM) closeout has been processed and is now available for your review.
```

You should see email body that follows the appropriate conditions in the Agility Ticket [B-21525](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21525)

___
# Screenshots of the email bodies
## Navy, Marine Corps, Coast Guard (AER)
<img width="704" alt="Screenshot 2024-11-01 at 11 02 09 AM" src="https://github.com/user-attachments/assets/d88a4f97-8326-4c09-a3de-e5c07441f48c">

## Nave, Marine Corps, Coast Guard (not AER)
<img width="696" alt="Screenshot 2024-11-01 at 10 59 49 AM" src="https://github.com/user-attachments/assets/dac7b84e-ceaa-4dea-b491-f545abdd4c1a">

## Army (AER)
<img width="715" alt="Screenshot 2024-11-01 at 11 09 29 AM" src="https://github.com/user-attachments/assets/df3ea7a8-8bbf-4ed8-867d-116f226c9c7a">

## Army (not AER)
<img width="693" alt="Screenshot 2024-11-01 at 11 07 31 AM" src="https://github.com/user-attachments/assets/a31798fa-f17d-47ec-9c5b-23932fc6fd7d">

## Air Force, Space Force (AER)
<img width="702" alt="Screenshot 2024-11-01 at 11 15 27 AM" src="https://github.com/user-attachments/assets/a550c0ce-d7ee-4dd3-af2a-1c8d525d425c">

## Air Force, Space Force (not AER)
<img width="702" alt="Screenshot 2024-11-01 at 11 16 43 AM" src="https://github.com/user-attachments/assets/f30b1d57-bb8b-4613-b878-2a3f85950d83">
